compiler/syntax/src/res_core.ml

```diff
@@ -4582,12 +4640,13 @@ and parse_field_declaration_region ?found_object_field p =
  *  | { field-decl, field-decl }
  *  | { field-decl, field-decl, field-decl, }
  *)
-and parse_record_declaration p =
+and parse_record_declaration ?current_type_name_path ?inline_types p =
   Parser.leave_breadcrumb p Grammar.RecordDecl;
   Parser.expect Lbrace p;
   let rows =
     parse_comma_delimited_region ~grammar:Grammar.RecordDecl ~closing:Rbrace
-      ~f:parse_field_declaration_region p
+      ~f:(parse_field_declaration_region ?current_type_name_path ?inline_types)
```

Comment:
Reanalyze gives here `Call must have named argument f` in CI.
One possibility is to remove the termination analysis as reanalyze won't be supported starting from ocaml 5.3 anyway. Simplifies the coding as one does not need to respect its syntactic restrictions, the downside is one needs to be careful about not introducing infinite loops.

Reply:
@cristianoc sounds reasonable to turn it off. Can I turn off just the termination analysis?

Reply:
That's the only thing used in tests, the termination analysis.
I think there's a single target in CI where it's active CC @cknitt